packages/datadog-plugin-cypress/src/cypress-plugin.js

```diff
@@ -321,7 +322,8 @@ class CypressPlugin {
     incrementCountMetric(name, {
       testLevel,
       testFramework: 'cypress',
-      isUnsupportedCIProvider: this.isUnsupportedCIProvider,
+      isUnsupportedCIProvider: !this.ciProviderName,
+      provider: this.ciProviderName,
```

Comment:
According to the RFC provider metric is not added on the `event_created` metric, but rather on a new metric `test_session`: 

> A nice to have feature is being able to collect the CI name of the supported CI list of a given test session in order to check the behavior of a given CI provider. Currently we only have the "is_unsupported_ci" boolean tag in the event_create metric, but we are not able to distinguish between supported providers. 

> At the same time the event_create metric along with the event_finish metric are the ones with the largest number of tags which makes them the metrics with the biggest cardinality.

> That's why the proposal is to include a new "test_session" metric with....

Reply:
great catch, thanks! That's the problem with diagonal reading 🤦